File: php/Disk.php
<?php

class Disk
{
    private $type, $capacity, $price;


    public function __construct($type = "-", $capacity = 0, $price = 0)
    {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }


    public function setType($type)
    {
        $this->type = $type;
    }

    public function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function getType()
    {
        return $this->type;
    }

    public function getCapacity()
    {
        return $this->capacity;
    }

    public function getPrice()
    {
        return $this->price;
    }
}

File: java/Disk.java
package latihan3.java;

/**
 * Disk
 */
public class Disk {

    private String type;
    private int capacity;
    private int price;

    public Disk(String type, int capacity, int price) {
        this.setType(type);
        this.setCapacity(capacity);
        this.setPrice(price);
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Disk() {
    }

}